background.js

```diff
@@ -68,7 +68,8 @@ async function githubIssueApi({ issue, path = '', data, method = 'GET' }) {
   const result = await response.json();
 
   if (!(await response).ok) {
-    throw new Error(result);
+    console.error("Github request failed", result);
```

Comment:
Good point - initially I just saw the error was "[object Object]" so added the log to move forward, then later decided it might as well be in the error but forgot to remove the log.